### ISSUE TITLE
IP packet dump to pcap log format

### ISSUE THREAD

**Issue by EdwardRyu** [espressif/esp-adf#1084]
Hello Espressif Support Team,

I wonder if you support a feature of capturing IP packets and converting into pcap format.
Or do you have any script to generate pcap log from your LwIP log?

Thanks,
Edward

**Comment by jason-mao** [comment]
@EdwardRyu I don't think it is useful and it's difficult. What is the reason for your requirements?

**Comment by EdwardRyu** [comment]
@jason-mao This is just a better-to-have, not a must-have. I just wanted to know if you support this as many major modem IC providers have this feature.
